src/ccs/confluent.sh

```diff
@@ -993,6 +993,17 @@ list_command() {
     fi
 }
 
+# CLI-84: If using Mac, version must be >= 10.13
+validate_os_version() {
+    local OS=$(uname -s)
```

Comment:
You shouldn't have to redo this, this is already available in https://github.com/confluentinc/confluent-cli/blob/master/src/ccs/confluent.sh#L24

Reply:
Ah, great!